src/Compilers/CSharp/Portable/Symbols/Source/SourceOrdinaryMethodSymbol.cs

```diff
@@ -194,7 +185,7 @@ protected override (TypeWithAnnotations ReturnType, ImmutableArray<ParameterSymb
                 forceMethodTypeParameters(returnType, this, declaredConstraints);
             }
 
-            return (returnType, parameters, _lazyIsVararg, declaredConstraints);
+            return (returnType, parameters, flags.IsVarArg, declaredConstraints);
```

Comment:
>flags.IsVarArg [](http://example.com/codeflow?start=44&length=14)

It looks like there is only one call site for ```MakeParametersAndBindReturnType```` and this value should be accessible directly where it is actually needed. I think we should remove it from the result tuple.

Reply:
Sure :)